src/main/scala/ch/openolitor/buchhaltung/reporting/RechnungReportData.scala

```diff
@@ -69,4 +77,70 @@ trait RechnungReportData extends AsyncConnectionPoolContextAware with Buchhaltun
       }
     }
   }
+
+  def createQrCode(rechnung: RechnungDetail, kontoDaten: KontoDaten, projekt: Projekt): String = {
+    val result = stammdatenReadRepository.getPersonen(rechnung.kunde.id) map { personen =>
+      var bill = new Bill();
+      val language = projekt.sprache match {
+        case Locale.FRENCH  => Bill.Language.FR;
+        case Locale.GERMAN  => Bill.Language.DE;
+        case Locale.ITALIAN => Bill.Language.IT;
+        case Locale.ENGLISH => Bill.Language.EN;
+        case _              => Bill.Language.DE;
+      }
+      bill.setLanguage(language)
+      kontoDaten.iban map { iban =>
+        bill.setAccount(iban);
+      }
+      bill.setAmount(rechnung.betrag.toDouble);
+      bill.setCurrency(projekt.waehrung.toString);
+
+      // Set creditor
+      val creditor = new Address();
+      creditor.setName(projekt.bezeichnung)
+      creditor.setStreet(projekt.strasse.getOrElse(""));
+      creditor.setHouseNo(projekt.hausNummer.getOrElse(""));
+      creditor.setPostalCode(projekt.plz.getOrElse(""));
+      creditor.setTown(projekt.ort.getOrElse(""));
+      creditor.setCountryCode("CH");
+      bill.setCreditor(creditor);
```

Comment:
Countrycode is fix on CH. Can't this also be used in Germany?

Reply:
No, this is only specific to switzerland. I am not sure if this will be compatible with the EU qrCode. We'll need to wait till the next revision in november...